boot/freeldr/freeldr/arch/i386/hwdisk.c

```diff
@@ -82,6 +82,13 @@ DiskOpen(CHAR* Path, OPENMODE OpenMode, ULONG* FileId)
     PARTITION_TABLE_ENTRY PartitionTableEntry;
     CHAR FileName[1];
 
+    if (DiskReadBufferSize == 0)
```

Comment:
`DiskReadBufferSize` is not used in this very function. Add a comment that this is early/once protecting `DiskRead()`?